feature-libs/product/configurators/rulebased/_index.scss

```diff
@@ -20,7 +20,8 @@ $configurator-common-components: cx-configurator-template,
   cx-configurator-conflict-suggestion, cx-configurator-conflict-description,
   cx-configurator-issues-notification,
   cx-configurator-overview-notification-banner, cx-configure-product,
-  cx-configure-cart-entry, cx-configurator-issues-notification !default;
+  cx-configure-cart-entry, cx-configurator-cart-entry-info,
```

Comment:
I also forgot about that, but we should move the common styles in a structure under /feature-libs/product/configurators/common/styles